src/octoprint/printer/standard.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, fileManager, analysisQueue, printerProfileManager):
 			except:
 				self._logger.exception("Error while processing analysis queues from {}".format(name))
 
+		#hook card upload
+		self.sd_card_upload_hooks = plugin_manager().get_hooks("octoprint.printer.cardupload")
```

Comment:
Not 100% yet about the name. `octoprint.printer.sd_card_upload`?

Reply:
ha yes, I did not pay attention.
I will correct that :)

Reply:
fixed :p